plio/views.py

```diff
@@ -261,11 +261,29 @@ def duplicate(self, request, uuid):
         # return 404 if user cannot access the object
         # else fetch the object
         plio = self.get_object()
+        items = list(Item.objects.filter(plio__id=plio.id))
+        questions = Question.objects.filter(item__plio__id=plio.id)
         # django will auto-generate the key when the key is set to None
         plio.pk = None
         plio.uuid = None
-        plio.status = "draft"  # a duplicated plio will always be in "draft" mode
+        plio.status = "draft"
+        # a duplicated plio will always be in "draft" mode
         plio.save()
+
+        if items:
+            for index, _ in enumerate(items):
+                items[index].plio = plio
+                items[index].pk = None
+
+            # create the items
+            items = Item.objects.bulk_create(items)
+        # questions part
+        if questions:
```

Comment:
questions with images haven't been handled as well?

Reply:
done in recent commit